common/djangoapps/student/tests/test_email.py

```diff
@@ -133,6 +141,17 @@ def _assert_activation_email(self, subject, body_fragments):
             self.assertIn(fragment, msg.body)
 
 
+    @mock.patch('student.tasks.log')
+    def test_send_email_to_inactive_user(self, mock_log):
+        """Verify that the activation email has been sent to un-activated user(logged-in via social-auth)"""
```

Comment:
Please follow the python guidelines for docstrings.
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings